CDP4WebServices.API/Services/Authorization/ObfuscationService.cs

```diff
@@ -53,13 +54,20 @@ namespace CDP4WebServices.API.Services.Authorization
     /// </summary>
     public class ObfuscationService : IObfuscationService
     {
+        /// <summary>
+        /// A cache of obfuscated things to be used to ensure <see cref="ModelLogEntry"/>s are cleaned up.
+        /// </summary>
+        private List<Guid> obfuscatedCache;
```

Comment:
perhaps you can make this a `HashSet` which will be a lot faster, just check what happens when you add the same item more than once (dunno if it fails or ignores)